probe-rs/src/debug/variable_cache.rs

```diff
@@ -429,6 +438,132 @@ impl VariableCache {
         };
         Ok(())
     }
+    /// Recursively process the deferred variables in the variable cache,
+    /// and add their children to the cache.
+    /// Enforce a max level, so that we don't recurse infinitely on circular references.
+    #[allow(clippy::too_many_arguments)]
+    pub fn recurse_deferred_variables(
+        &mut self,
+        debug_info: &DebugInfo,
+        memory: &mut dyn MemoryInterface,
+        parent_variable: Option<&mut Variable>,
+        registers: &DebugRegisters,
+        frame_base: Option<u64>,
+        _max_recursion_depth: usize,
```

Comment:
Why is there an unused argument here?

Reply:
I forgot to use it, in place of the hardcoded '10' ... consider it fixed :)